README.md

```diff
@@ -19,6 +19,7 @@ Download the official installer for your operating system:
  - [macOS](https://central.github.com/deployments/desktop/desktop/latest/darwin)
  - [Windows](https://central.github.com/deployments/desktop/desktop/latest/win32)
  - [Windows machine-wide install](https://central.github.com/deployments/desktop/desktop/latest/win32?format=msi)
+ - [Linux](https://central.github.com/deployments/desktop/desktop/latest/linux)
```

Comment:
Again, these URLs won't work currently because we haven't taught our release server about these.

This URL doesn't consider which package format you want to download.